tests/datasource/test_new_datasource_with_sql_data_connector.py

```diff
@@ -891,7 +891,7 @@ def test_batch_request_sql_with_schema(
         "runtime_parameters": {"query": "SELECT * FROM main.table_2"},
         "batch_identifiers": {
             "pipeline_stage_name": "core_processing",
-            "airflow_run_id": 1234567890,
+            "airflow_run_id": 1234567891,
```

Comment:
Non-blocking, just curious : why the `0` -> `1`?

Reply:
@Shinnnyshinshin This is critical -- thank you for drawing attention to this.  We must disambiguate `Batch` ID -- otherwise, multi-batch scenarios will fail (batch metric service will see that the value exists and return it, instead of computing new one).  So this is a future-proofing hygiene step.